### PR TITLE
ChangeLog for 1.6.1

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+* #932: Write config file when channel is removed
+* Remove "-" between "ZNC" and version number
+* #946: Fix rare conflict of HTTP-Basic auth and cookies
+* Update error message about --datadir
+* #895: Main ZNC IRC channel is on freenode
+* Add non-minified jQuery's source
+* #954: Fix startup failure when simple_away is loaded after awaynick
+* #956: configure.ac: specify required compiler versions
+* #944: Don't print local ip address on 404 page
+* #967: Fix query buffer size for users who have 0 in default channel buffer size
```
Comment (DarthGandalf) [inline ChangeLog.md]: Let's sort them by importance?

File: ChangeLog.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
+# ZNC 1.6.1 (2015-MM-DD)
+
+## Fixes
+
+* Fix sublists in the changelog
```
Comment (DarthGandalf) [inline ChangeLog.md]: Does this even needs to be listed here?

File: ChangeLog.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+* sasl:
+    * #875: fix help text for requireauth
+    * update help for "set"
+
+* savebuff:
+    * #868: fix timer initialization
+
+* webadmin:
+    * #872: webadmin checkboxes
+    * Explain why encoding settings are disabled
```
Comment (DarthGandalf) [inline ChangeLog.md]: if they are disabled

**Comment by DarthGandalf** [comment]
Generally, I think it's easier to edit it on wiki first, and then export to ChangeLog.md

**Comment by jpnurmi** [comment]
> Generally, I think it's easier to edit it on wiki first, and then export to ChangeLog.md

Yeah, sounds better. Let's continue at http://wiki.znc.in/ChangeLog/1.6.1 /cc @Mikaela

File: ChangeLog.md (hunk 2 above)
Comment (Mikaela) [inline ChangeLog.md]: I don't think so.

**Pull request closed without merging**